examples/controls_example/public/react_controls/data_controls/data_control_editor.test.tsx

```diff
@@ -64,20 +64,21 @@ const controlGroupApi = {
   grow: new BehaviorSubject(DEFAULT_CONTROL_GROW),
   width: new BehaviorSubject(DEFAULT_CONTROL_WIDTH),
 } as unknown as ControlGroupApi;
+const onSave = jest.fn();
```

Comment:
All the changes in this file can also be reverted

Reply:
Woops - missed that, thanks. Done in ce2897f2f023b5d5ec54363c8eae75e974ac9c07